userguide/getting-started/quick_start_with_kb_api.adoc

```diff
@@ -1,16 +1,28 @@
-Now that you have a system setup, we will go through a small tutorial to get started using the system. 
+= Quick Start with the Kill Bill API
 
-At the end of this tutorial, you will have created a Kill Bill tenant and configured such tenant with a valid catalog. The tenant represents a logical deployment running on a (set of) physical servers. For more information about tenants, refer to https://killbill.github.io/slate/#tenant[this section]. The tenant configuration typically includes all the specifics of your business, e.g what you sell, how you sell it, ...
+This guide explains how to get familiar with the Kill Bill API by performing the following tasks:  
 
-You will have also created an `Account` (a consumer), set up this consumer with a valid payment method, subscribed her to one of your product, and finally will be able to see the invoice and payment generated by the system.
+. <<step1, Create an account>> 
+. <<step2, Add a payment method to the account>>
+. <<step3, Set up a subscription for the account>>
 
-=== API Client
+== Before You Begin
 
-Now that you are familiar with the basics, the next step is to integrate Kill Bill in your application using our APIs. Our https://killbill.github.io/slate/[API documentation] contains snippets to help you get started.
+. Make sure you have installed Kill Bill and Kaui and launched them per the https://docs.killbill.io/latest/getting_started.html.html[_Getting Started_] guide. 
+. Follow the _Quick Start with Kaui_ guide to https://docs.killbill.io/latest/quick_start_with_kaui.html#step1[create your first tenant] and a https://docs.killbill.io/latest/quick_start_with_kaui.html#step2[simple catalog].
 
-> Note: This section assumes you are already familiar with the core concepts of Kill Bill. If you aren't, make sure to go back to the previous section first.
+== Additional Resources
 
-==== Creating Your First Account
+* https://killbill.github.io/slate[_API Reference_]
+
+* https://docs.killbill.io/latest/userguide_kaui.html[_Kaui Guide_] 
+
+* https://docs.killbill.io/latest/Kill-Bill-Glossary.html[Kill Bill Glossary]
+ 
+[step1]
+== Create an Account
+
+In this section, we will create an https://docs.killbill.io/latest/Kill-Bill-Glossary.html#account[account^] for a customer, which stores the data your organization uses to transact business with a customer. To keep things simple, we will create an account with a minimum of information.
 
 ++++
```

Comment:
Would it also help to include the response for each API endpoint? You can get the response from the slate docs (By checking the "Example Response" section for each programming language under the corresponding endpoint)

Reply:
Done